### PR TITLE
Updates for 0.10-compatible libs

### PR DESCRIPTION
**Pull request by nwolverson**

### CODE REVIEW

File: src/Psa/Output.purs [hunk 1]
```diff
@@ -8,16 +8,16 @@ module Psa.Output
...
-import Data.Array as Array
+import Data.String as Str
 import Node.Path as Path
-import Psa.Types (PsaOptions, PsaError, PsaAnnotedError, PsaPath(..), PsaResult, Position, Filename, Lines,
-                  compareByLocation)
+import Data.Foldable (foldl, any)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Data.String (Pattern(Pattern))
+import Data.Tuple (Tuple(..), fst)
+import Psa.Types (PsaOptions, PsaError, PsaAnnotedError, PsaPath(..), PsaResult, Position, Filename, Lines, compareByLocation)
```
Comment (natefaubion) [inline src/Psa/Output.purs:19]: The imports here are kind of weird. Can we sort them? Like, Node.Path is in the middle of a bunch of Data imports.

File: src/Psa/Output.purs [hunk 2]
```diff
@@ -155,15 +155,15 @@ shouldShowError { filterCodes, censorCodes, censorSrc, censorLib, censorWarnings
 
 errorPath :: Array String -> String -> String -> PsaPath
 errorPath libDirs path short =
-  if any (path `startsWith` _) libDirs
+  if any (\dir -> path `startsWith` Pattern dir) libDirs
```
Comment (natefaubion) [inline src/Psa/Output.purs]: Since we've already imported `Str`, can we just use `Str.Pattern` like you use in other modules?

**Review by natefaubion — CHANGES_REQUESTED** [review]

File: src/Psa/Output.purs (hunk 1 above)
Comment (nwolverson) [reply src/Psa/Output.purs:19]: Ah, I let psc-ide sort these, which puts qualified imports at the top, i can tweak

Comment (natefaubion) [reply src/Psa/Output.purs:19]: I find this rule make very little sense now that we've gotten rid of `qualified` 😛

Comment (nwolverson) [reply src/Psa/Output.purs:19]: I see your point. @kRITZCREEK maybe worth noticing if we see a consensus on this?

**Review by natefaubion — APPROVED** [review]

File: src/Psa/Output.purs (hunk 1 above)
Comment (kritzcreek) [reply src/Psa/Output.purs:19]: The idea is that the qualified and implicit imports are the one you edit by hand so I put them first.

**Pull request merged**